src/main/java/gregtech/api/block/BlockCustomParticle.java

```diff
@@ -1,9 +1,9 @@
 package gregtech.api.block;
 
 import codechicken.lib.vec.Vector3;
-import gregtech.api.net.NetworkHandler;
-import gregtech.api.net.NetworkUtils;
-import gregtech.api.net.packets.SPacketBlockParticle;
+import gregtech.api.GregTechAPI;
+import gregtech.core.network.NetworkUtils;
```

Comment:
Incorrect order of imports. All committed files should be checked. I think you can set IDEA to do this automatically.